packages/profile-sync-controller/src/controllers/user-storage/contact-syncing/types.ts

```diff
@@ -0,0 +1,40 @@
+import type { Hex } from '@metamask/utils';
+
+import type {
+  USER_STORAGE_VERSION_KEY,
+  USER_STORAGE_VERSION,
+} from './constants';
+import type { UserStorageControllerMessenger } from '../UserStorageController';
+import type UserStorageController from '../UserStorageController';
+
+export type UserStorageContactEntry = {
+  /**
+   * The Version 'v' of the User Storage.
+   * NOTE - will allow us to support upgrade/downgrades in the future
+   */
+  [USER_STORAGE_VERSION_KEY]: typeof USER_STORAGE_VERSION;
+  /** the address 'a' of the contact */
+  a: string;
+  /** the name 'n' of the contact */
+  n: string;
+  /** the chainId 'c' of the contact */
+  c: Hex;
```

Comment:
I missed this earlier. We'll have to adapt the chainId for non-EVM situations.
Not a blocker for now, but we might need conversions later